charts/add-ons/dashboard/templates/web.yaml

```diff
@@ -68,12 +68,10 @@ spec:
         - -api-addr=linkerd-controller-api.{{.Values.global.namespace}}.svc.{{.Values.global.clusterDomain}}:8085
         {{- if .Values.global.grafanaUrl }}
         - -grafana-addr={{.Values.global.grafanaUrl}}
-        {{- else if .Values.grafana.enabled }}
-        - -grafana-addr={{.Values.grafana.name}}.{{.Values.global.namespace}}.svc.{{.Values.global.clusterDomain}}:3000
-        {{- end}}
-        {{- if .Values.tracing.enabled }}
-        - -jaeger-addr={{.Values.tracing.jaeger.name}}.{{.Values.global.namespace}}.svc.{{.Values.global.clusterDomain}}:16686
+        {{- else}}
+        - -grafana-addr=linkerd-grafana.{{.Values.global.namespace}}.svc.{{.Values.global.clusterDomain}}:3000
         {{- end}}
+        - -jaeger-addr=linkerd-jaeger.{{.Values.global.namespace}}.svc.{{.Values.global.clusterDomain}}:16686
```

Comment:
but as we are removing the condition and passing that variable all the time, This means that users will see the `jaeger` icon irrespective of the add-on being installed which is not great. wondering if there's a better way to do this. :thinking: 

@alpeb

Reply:
This stems from the inability of add-ons to access each other's Helm configs, right?
The web component currently mounts `linkerd-config`; we could have it mount `linkerd-config-addons` as well, and override the grafana/jaeger flags based off of that.

Reply:
@alpeb Great Suggestion!
Updated the PR regarding the same!